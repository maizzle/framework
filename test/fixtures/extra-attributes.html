<div>Test</div>

<div role="article">Test</div>
