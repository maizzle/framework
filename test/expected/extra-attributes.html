<div role="article" class="text-center">Test</div>

<div role="article" class="text-center">Test</div>
